recipe/meta.yaml

```diff
@@ -9,57 +9,114 @@ source:
   - url: https://bitbucket.org/osrf/{{ name }}/get/{{ name }}{{ version.split('.')[0] }}_{{ version }}.tar.gz
     sha256: f1c873dfa17c1653332321ef4f10430fb22825de0688017455a2983f176cd69e
     patches:
-      - use-external-libs-config.patch
-      - qwt-fix.patch
-      - normalize-ogre-path.patch
+      - fix_build.patch  # [linux]
+      - use-external-libs-config.patch  # [win]
+      - qwt-fix.patch  # [win]
+      - normalize-ogre-path.patch  # [win]
 
 build:
-  number: 4
-  skip: true  # [not win]
+  number: 0
```

Comment:
Can we just increase the number to `5` and make the skip only on `[not osx]`? (and can you make this pull request editable too? I can also help on the cleanup.)

Reply:
this is gazebo 11 so I am not sure it will build on Windows, unfortunately, and since it's a completely new version number I changed the build number back to 0.

Reply:
Oh, wait, you also have gazebo 11 :man_facepalming: 

Then I will have to do your changes. sry about that.

Reply:
ok let's hope this builds :)